### PR TITLE
INFINITY-2378/2820 Collect all relevant task logs from test, reduce noise

### PR DESCRIPTION
**Pull request by nickbp**
- Get task ids before and after a test, then collect logs from the tasks that appeared while a failed test was running. Print progress as log collection is performed so that the user knows what's happening.
- Turn down some noise from underlying libraries by setting them to WARNING.
- Emit all test logs to stdout instead of a mixture of stdout+stderr. This gets around teamcity printing the two separately.
- Broaden usage of logging plan state when a test fails.

### CODE REVIEW

**Comment by mbdcos** [comment]
LGTM

File: testing/sdk_plan.py [hunk 1]
```diff
@@ -25,6 +27,16 @@ def get_recovery_plan(service_name):
...
 
 
+def list_plans(service_name):
+    def fn():
+        output = sdk_api.get(service_name, '/v1/plans')
+        try:
+            return output.json()
+        except:
+            return False
+    return shakedown.wait_for(fn)
```
Comment (NimaVaziri) [inline testing/sdk_plan.py]: Maybe use the `retry` library as we're gradually reducing dependency on shakedown?

File: frameworks/cassandra/tests/conftest.py [hunk 2]
```diff
@@ -1,37 +1,21 @@
...
-
 import pytest
+import sdk_plan
 import sdk_repository
 import sdk_security
-import sdk_utils
-import shakedown
 
-log = logging.getLogger(__name__)
+SERVICE_NAME = 'cassandra'
```
Comment (NimaVaziri) [inline frameworks/cassandra/tests/conftest.py]: We usually define `SERVICE_NAME` in [`config.py`](https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/tests/config.py#L16) for the services so perhaps use/import that for convergence's sake?

Comment (nickbp) [reply frameworks/cassandra/tests/conftest.py]: i tried consuming from there directly, but `import config` failed

Comment (NimaVaziri) [reply frameworks/cassandra/tests/conftest.py]: ah, might need to specify directory i.e. `from tests import config` as in https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/tests/test_sanity.py#L15

Comment (nickbp) [reply frameworks/cassandra/tests/conftest.py]: aha, yep, forgot about that

**Pull request merged**